src/main/resources/templates/sites/ranking.html

```diff
@@ -76,8 +76,8 @@ <h2 th:text="#{sites.ranking}"></h2>
 </div>
 <!-- User List -->
 <div class="container-fluid">
-  <div class="shadow-sm" id="table-container" th:if="${allUsers.size()>0}">
-    <table class="user-table table">
+  <div th:if="${allUsers.size()>0}" style="overflow-x:auto;">
+    <table style="width: 80%; margin-left: 10%; border-color: #332f2d" class="user-table table shadow-sm">
```

Comment:
![Unbenannt_2](https://user-images.githubusercontent.com/43335130/54420616-c44aeb00-470a-11e9-9775-2595e83f6c9a.PNG)

I'm also not shure if scrollbar fixes the responsiveness of the table. I wouldn't want to scroll horizontaly to see a table. We should look for another solution (scaling the column gabs and font-size and also leaving out the column description seems like a better solution to me)